docs/end_to_end_tutorials/dev_practices/component_wise_evaluation.md

```diff
@@ -0,0 +1,41 @@
+# Component Wise Evaluation
+## Evaluating Retrieval
+
+### BEIR dataset
+
+BEIR is useful for benchmarking if a particular retrieval model generalize well to niche domains in a zero-shot setting (without fine-tuning).
+
+Here is a notebook showing how the BEIR dataset can be used with your retrieval pipeline.
+
+We will be adding more methods to evaluate retrieval soon. This includes evaluating retrieval on your own dataset.
+
+## Evaluating the Query Engine
```

Comment:
maybe helps to provide some context on what a query engine is and what are inputs/outputs to give users a sense of what they're evaluating. 

also did you mean evaluating the query engine or evaluating the LLM? to me evaluating the query engine feels like e2e evaluation

Reply:
I clarified that this is specifically where we don't evaluate the retrieval step.